### ISSUE TITLE
layers misplaced

### ISSUE THREAD

**Issue by sasha567-vng** [HashLips/hashlips_art_engine#761]
even though i have place my layers  in correct order it is generating opposite of it.!! i have tried re arranging the layers again but still it keeps generating the images with wrong layers. please help

**Comment by bolshoytoster** [comment]
@sasha567-vng can you paste the `layerConfigurations` bit from your `src/config.js` please. You might just have to change the order of the layers in that.

**Comment by bolshoytoster** [comment]
@nolimitf8t that should be correct, it should put `Background`at the back and `Mouth` at the front.

**Comment by sasha567-vng** [comment]
the problem is something.! even though my layers are placed in correct order by the time of generating it is not following the orders that i have made inside the code.! help please

**Comment by nolimitf8t** [comment]
exactly, They are placed in the right order but still keep generating misplaced.

**Comment by bolshoytoster** [comment]
@nolimitf8t Could you send one please?

**Comment by sasha567-vng** [comment]
bro the thing is i want to generate my own custom image not the original hashlips one, before i generate my art i  tried haslips build in art and ran it.! it had no problem all the images are generated correctly. ok but when i added my own custom layers and even though i have putted my layers orders in correct way it keeps generating in wrong way.! the generated art is not following my layers
